.github/workflows/reusable-build.yml

```diff
@@ -54,19 +34,17 @@ jobs:
       - name: Matrix Variables
         shell: bash
         env:
-          MATRIX_SOURCE_IMAGE: ${{ matrix.source_image }}
-          MATRIX_IMAGE_NAME: ${{ matrix.image_name }}
+          IMAGE_NAME: ${{ inputs.image_name }}
+          FEDORA_VERSION: ${{ inputs.fedora_version }}
+          SOURCE_IMAGE_NAME: ${{ inputs.source_image_name }}
         run: |
-          if [[ -z $MATRIX_SOURCE_IMAGE ]]; then
-              echo "MATRIX_SOURCE_IMAGE empty"
-              SOURCE_IMAGE=$MATRIX_IMAGE_NAME
-          else
-              echo "MATRIX_SOURCE_IMAGE not empty"
-              SOURCE_IMAGE=$MATRIX_SOURCE_IMAGE
+          SOURCE_IMAGE=$IMAGE_NAME
+          if [[ "$SOURCE_IMAGE_NAME" != "" ]]; then
+            SOURCE_IMAGE=$SOURCE_IMAGE_NAME
           fi
 
           echo "SOURCE_ORG=fedora-ostree-desktops" >> $GITHUB_ENV
-          echo "IMAGE_NAME=$MATRIX_IMAGE_NAME-main" >> $GITHUB_ENV
+          echo "IMAGE_NAME=$IMAGE_NAME-main" >> $GITHUB_ENV
           echo "SOURCE_IMAGE=$SOURCE_IMAGE" >> $GITHUB_ENV
```

Comment:
This can be removed when F41 is EOL'd.  It's needed so we can keep the old "sericea" etc  naming convention for the existing images